### ISSUE TITLE
"undefined: 0" error

### ISSUE THREAD

**Issue by davidgilbertson** [sasstools/sass-lint#212]
I'm getting the following issue on several different machines (mac and linux) since upgrading to `1.2.1` when I have no failing tests:
```
undefined:0


SyntaxError: Unexpected end of input
    at Object.parse (native)
    at Function.sassLint.resultCount (/Users/davidg/www/benefit-site/node_modules/sass-lint/index.js:23:26)
    at Function.sassLint.outputResults (/Users/davidg/www/benefit-site/node_modules/sass-lint/index.js:110:12)
    at detectPattern (/Users/davidg/www/benefit-site/node_modules/sass-lint/bin/sass-lint.js:21:10)
    at Object.<anonymous> (/Users/davidg/www/benefit-site/node_modules/sass-lint/bin/sass-lint.js:54:3)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```
I believe it's because on [this line](https://github.com/sasstools/sass-lint/pull/170/files#diff-168726dbe96b3ce427e7fedce31bb0bcR23), the code is trying to `JSON.parse()` an empty string which throws.

However, I'm also getting an error when I do have failing tests, where the value of `results` is something like this: `\n\u001b[4mapp/components/registrationForm/RegistrationForm/RegistrationForm.scss\u001b[24m\n  \u001b[90m15:2\u001b[39m  \u001b[33mwarning\u001b[39m  Space expected between blocks  \u001b[90mempty-line-between-blocks\u001b[39m\n  \u001b[90m18:2\u001b[39m  \u001b[33mwarning\u001b[39m  Space expected between blocks  \u001b[90mempty-line-between-blocks\u001b[39m\n  \u001b[90m22:2\u001b[39m  \u001b[33mwarning\u001b[39m  Space expected between blocks  \u001b[90mempty-line-between-blocks\u001b[39m\n\n\u001b[33m\u001b[1m✖ 3 problems (0 errors, 3 warnings)\n\u001b[22m\u001b[39m` which is also not valid JSON.

**Comment by DanPurdy** [comment]
Hey David,
Thanks for the report.

Yeah i believe this is related #197 I believe. I'm looking into it as we speak.

**Comment by davidgilbertson** [comment]
Cool, thanks Dan.

**Comment by DanPurdy** [comment]
@davidgilbertson we've released `1.2.2` with a small fix for this. Will close this issue but if you have any problems let me know.
